gtwrap/interface_parser.py

```diff
@@ -730,24 +775,26 @@ class GlobalFunction:
     Rule to parse functions defined in the global scope.
     """
     rule = (
-        ReturnType.rule("return_type")  #
+        Optional(Template.rule("template"))
+        + ReturnType.rule("return_type")  #
         + IDENT("name")  #
         + LPAREN  #
         + ArgumentList.rule("args_list")  #
         + RPAREN  #
         + SEMI_COLON  #
-    ).setParseAction(
-        lambda t: GlobalFunction(t.name, t.return_type, t.args_list))
+    ).setParseAction(lambda t: GlobalFunction(t.name, t.return_type, t.
+                                              args_list, t.template))
 
     def __init__(self,
                  name: str,
                  return_type: ReturnType,
                  args_list: ArgumentList,
+                 template: Template,
                  parent: str = ''):
         self.name = name
         self.return_type = return_type
         self.args = args_list
-        self.is_const = None
+        self.template = template
```

Comment:
What happened to const? Deciding to no longer keep track of const is a different PR/decision (with pros and cons)

Reply:
It's never used in this class, isn't initialized in the constructor and a global function with a const at the end aka `void GlobalFunc() const;` is something I've never seen.